CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v6.1.0 (02/24/2021)
+
+#### Enhancements:
```

Comment:
Please change `Enhancements` to `Features`

Reply:
👍